src/wallet/wallet.cpp

```diff
@@ -4365,7 +4383,26 @@ int CMerkleTx::GetBlocksToMaturity() const
 }
 
 
-bool CMerkleTx::AcceptToMemoryPool(const CAmount& nAbsurdFee, CValidationState& state)
+bool CWalletTx::AcceptToMemoryPool(const CAmount& nAbsurdFee, CValidationState& state)
 {
-    return ::AcceptToMemoryPool(mempool, state, tx, true, nullptr, nullptr, false, nAbsurdFee);
+    // We must set fInMempool here - while it will be re-set to true by the
+    // entered-mempool callback, if we did not there would be a race where a
+    // user could call sendmoney in a loop and hit spurious out of funds errors
+    // because we think that the transaction they just generated's change is
+    // unavailable as we're not yet aware its in mempool.
+    bool ret = ::AcceptToMemoryPool(mempool, state, tx, true, nullptr, nullptr, false, nAbsurdFee);
+    fInMempool = ret;
+    return ret;
+}
+
+bool CalculateEstimateType(FeeEstimateMode mode, bool opt_in_rbf) {
```

Comment:
In commit "Use callbacks to cache whether wallet transactions are in mempool"

Looks like you accidentally resurrected this CalculateEstimateType function in the rebase. Should remove.